src/main/java/com/madikhan/app/util/converter/BirthdayConverter.java

```diff
@@ -0,0 +1,25 @@
+package com.madikhan.app.util.converter;
+
+import com.madikhan.app.model.Birthday;
+
+import javax.persistence.AttributeConverter;
+import java.sql.Date;
+import java.util.Optional;
+
+public class BirthdayConverter implements AttributeConverter<Birthday, Date> {
+    @Override
+    public Date convertToDatabaseColumn(Birthday birthdayEntity) {
+        return Optional.ofNullable(birthdayEntity)
+                .map(Birthday::getBirthDate)
+                .map(Date::valueOf)
+                .orElse(null);
```

Comment:
it's really bad practice to return null, worse only return null in Optional.orElseGet